### PR TITLE
Group task setup backend repo

### PR DESCRIPTION
**Pull request by MarcoDiaz2000**
-  Setup backend repo
-  Use Postgres as your database
-  Add Gems and Setup the BackEnd
-  Add linters
-  Update readme file to include all colaborators
-  Enable CORS

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
```
Comment (Salimer) [inline README.md:3]: Please add the project title at the beginning of the Read Me file.

**Review by Salimer — CHANGES_REQUESTED** [review]
## Hello @MarcoDiaz2000 
Good job so far in implementing the backend repo. However, there are some required changes you need to include.

## Highlights:

- [x] Professional Read Me file
- [x] Correct linters
- [x] Postgres is used as the database

## Required Changes

- [ ] Remove the devise gem as we don't need it per requirements. We might add it later if we have extra time before evaluation time.
- [ ] Update CORS Configuration: Ensure that your CORS configuration in `config/initializers/cors.rb` is correct to enable CORS as per requirements.
- [ ] Check comments

**Review by Salimer — APPROVED** [review]
Hello @MarcoDiaz2000 , good job on implementing the changes. Your PR is approved and there is nothing else to say other than; **it's time to merge**

**Comment by MarcoDiaz2000** [comment]
Thanks, @Salimer

**Pull request merged**